src/uwtools/tests/test_rocoto.py

```diff
@@ -52,3 +53,17 @@ def test_write_rocoto_xml(tmp_path):
 
     expected = support.fixture_path("hello_workflow.xml")
     support.compare_files(expected, output)
+
+
+def test_rocoto_xml_is_valid():
+    with resources.as_file(resources.files("uwtools.resources")) as resc:
+        with open(resc / "schema_with_metatasks.rng", "r", encoding="utf-8") as f:
+            schema = etree.RelaxNG(etree.parse(f))
+
+    valid_xml = support.fixture_path("hello_workflow.xml")
+    tree = etree.parse(valid_xml)
+    assert schema.validate(tree)
+
+    invalid_xml = support.fixture_path("rocoto_invalid.xml")
+    tree = etree.parse(invalid_xml)
+    assert schema.validate(tree) is False
```

Comment:
Does it make sense to simplify this line with:

```
assert not schema.validate(tree
```

Reply:
This can be DRYed out with `parametrize`:
```
@pytest.mark.parametrize("vals", [("hello_workflow.xml", True), ("rocoto_invalid.xml", False)]
def test_rocoto_xml_is_valid(vals):
    fn, validity = vals
    # same "with" block...
    xml = support.fixture_path(fn)
    tree = etree.parse(xml)
    assert schema.validate(tree) is validity
```

Reply:
Excellent suggestion! It made the code less repetitive.